roboto.js

```diff
@@ -1,155 +1,205 @@
-}
-@font-face {
-font-family: "Roboto";
-src: url(fonts/roboto/Roboto-BlackItalic.ttf) format("truetype");
-font-weight: 900;
-font-style: italic;
-}
-@font-face {
-font-family: "Roboto Mono";
-src: url(fonts/robotomono/RobotoMono-Thin.ttf) format("truetype");
-font-weight: 100;
-font-style: normal;
-}
-@font-face {
-font-family: "Roboto Mono";
-src: url(fonts/robotomono/RobotoMono-ThinItalic.ttf) format("truetype");
-font-weight: 100;
-font-style: italic;
-}
-@font-face {
-font-family: "Roboto Mono";
-src: url(fonts/robotomono/RobotoMono-Light.ttf) format("truetype");
-font-weight: 300;
-font-style: normal;
-}
-@font-face {
-font-family: "Roboto Mono";
-src: url(fonts/robotomono/RobotoMono-LightItalic.ttf) format("truetype");
-font-weight: 300;
-font-style: italic;
-}
-@font-face {
-font-family: "Roboto Mono";
-src: url(fonts/robotomono/RobotoMono-Regular.ttf) format("truetype");
-font-weight: 400;
-font-style: normal;
-}
-@font-face {
-font-family: "Roboto Mono";
-src: url(fonts/robotomono/RobotoMono-Italic.ttf) format("truetype");
-font-weight: 400;
-font-style: italic;
-}
-@font-face {
-font-family: "Roboto Mono";
-src: url(fonts/robotomono/RobotoMono-Medium.ttf) format("truetype");
-font-weight: 500;
-font-style: normal;
-}
-@font-face {
-font-family: "Roboto Mono";
-src: url(fonts/robotomono/RobotoMono-MediumItalic.ttf) format("truetype");
-font-weight: 500;
-font-style: italic;
-}
-@font-face {
-font-family: "Roboto Mono";
-src: url(fonts/robotomono/RobotoMono-Bold.ttf) format("truetype");
-font-weight: 700;
-font-style: normal;
-}
-@font-face {
-font-family: "Roboto Mono";
-src: url(fonts/robotomono/RobotoMono-BoldItalic.ttf) format("truetype");
-font-weight: 700;
-font-style: italic;
-}
-</style>`;
-
-document.head.appendChild($_documentContainer.content);
-
 /**
 @license
 Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
-This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
-The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
-The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
-Code distributed by Google as part of the polymer project is also
-subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
-*/
-/*
-  FIXME(polymer-modulizer): the above comments were extracted
-  from HTML and may be out of place here. Review them and
-  then delete this comment!
+This code may only be used under the BSD style license found at
+http://polymer.github.io/LICENSE.txt The complete set of authors may be found at
+http://polymer.github.io/AUTHORS.txt The complete set of contributors may be
+found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by Google as
+part of the polymer project is also subject to an additional IP rights grant
+found at http://polymer.github.io/PATENTS.txt
 */
-;
+
+// modified version of html function so that i can insert import meta
```

Comment:
why is something special needed because you used import.meta?

Reply:
I simply just wanted a function called `html` for syntax highlighting. the one in the polymer lib only accepts special types of inserted values that are what polymer calls literals and will exception if you just pass it a string.

Reply:
Ohh I see, that's a security thing. I'd say just do:

```js
const style = document.createElement('style');
style.innerText = ` ... `;
document.head.appendChild(style);
```

and don't worry about syntax highlighting. There's also no point making a template node.